source/material_model/latent_heat.cc

```diff
@@ -208,6 +208,7 @@ namespace aspect
             // this means, that there are no actual density "jumps", but gradual
             // transition between the materials
             double phase_dependence = 0.0;
+            double viscosity_phase_dependence = 1.0;
```

Comment:
Also there is a comment in line 181 saying that this part is only computing density, maybe extend that to mention also the phase dependence of the viscosity